curl http://$INGRESS/wrongpath
404 page not found

 echo '{"path":"/wrongpath","user_agent":"curl/7.64.1","response_flags":"-","duration":10,"downstream_remote_address":"10.128.0.125:63878","method":"GET","route_name":null,"request_id":"0eec59d8-addc-9ffe-aa5a-5170f928a334","response_code_details":"via_upstream","upstream_cluster":"outbound|80||frontend.online-boutique.svc.cluster.local","response_code":404,"upstream_local_address":"10.124.1.12:33076","connection_termination_details":null,"downstream_local_address":"10.124.1.12:8080","bytes_sent":19,"protocol":"HTTP/1.1","x_forwarded_for":"10.128.0.125","upstream_host":"10.124.1.14:8080","bytes_received":0,"authority":"35.202.35.23","upstream_transport_failure_reason":null,"start_time":"2021-05-22T11:32:44.284Z","upstream_service_time":"10","requested_server_name":null}'| jq .
{
  "path": "/wrongpath",
  "user_agent": "curl/7.64.1",
  "response_flags": "-",
  "duration": 10,
  "downstream_remote_address": "10.128.0.125:63878",
  "method": "GET",
  "route_name": null,
  "request_id": "0eec59d8-addc-9ffe-aa5a-5170f928a334",
  "response_code_details": "via_upstream",
  "upstream_cluster": "outbound|80||frontend.online-boutique.svc.cluster.local",
  "response_code": 404,
  "upstream_local_address": "10.124.1.12:33076",
  "connection_termination_details": null,
  "downstream_local_address": "10.124.1.12:8080",
  "bytes_sent": 19,
  "protocol": "HTTP/1.1",
  "x_forwarded_for": "10.128.0.125",
  "upstream_host": "10.124.1.14:8080",
  "bytes_received": 0,
  "authority": "35.202.35.23",
  "upstream_transport_failure_reason": null,
  "start_time": "2021-05-22T11:32:44.284Z",
  "upstream_service_time": "10",
  "requested_server_name": null
}

based on the upstream cluster I think it was proxied
And I can see /wrongpath in logs of istio-proxy and frontend containers
